node-src/ui/messages/errors/e2eBuildFailed.ts

```diff
@@ -0,0 +1,17 @@
+import chalk from 'chalk';
+import { dedent } from 'ts-dedent';
+import { error } from '../../components/icons';
+
+export default ({
+  flag,
+  errorMessage,
+}: {
+  flag: string;
+  errorMessage: string;
+}) => {
+  return dedent(chalk`
+      ${error} Failed to run \`chromatic --${flag}\`:
+      
+      ${errorMessage}
```

Comment:
We could add a list of causes for this error, or maybe better yet link to a troubleshooting page in our docs.

Reply:
I guess the main one we are worried about is when an error like `no such binary exists "build-archive-storybook"`. For a user that's going to be a pretty opaque error if they have no idea about what archive SB even is.